google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineRuntimeInstance.java

```diff
@@ -132,11 +139,18 @@ public void computeDeployments(@NotNull ComputeDeploymentsCallback callback) {
   public void disconnect() {
     // kill any executing deployment actions
     synchronized (createdDeployments) {
-      for (CancellableRunnable runnable : createdDeployments) {
+      for (CancellableRunnable runnable : createdDeployments.values()) {
         runnable.cancel();
       }
       createdDeployments.clear();
     }
+
+    createdDeployments
+        .keys()
+        .forEach(
+            project ->
+                ProjectManager.getInstance()
```

Comment:
Haha whoops, good call @chanseokoh. And @etanshaul, I'm not sure. Is there any way to check that? During deployment, is there something I can monitor (e.g. underlying process) that would close correctly if the platform was doing this?